backend/src/api.py

```diff
@@ -259,26 +261,72 @@ def contribute_review_container_logs(
 
 
 @app.get("/frontend/review/random")
-def frontend_review_random() -> FeedbackSchema:
-    if os.environ.get("ENV") == "production":
-        raise NotImplementedError("Reviewing is not ready yet")
-
+def frontend_review_random():
```

Comment:
no type?

Reply:
The response structure is checked at the return statement with feedback schema - there's just added `id` for the reference and I didn't want to add another unicorn schema to `schema.py` since this will disappear with database creation and simply the structure is parsed - just the `id` is added at the end.